src/components/Header.tsx

```diff
@@ -0,0 +1,29 @@
+export const Header = () => {
+  return (
+    <nav className="nav flex justify-between items-center w-full bg-[#90E9EF] text-black p-4">
+      <div className="flex items-center space-x-4">
+        <a href="/" className="site-logo hover:text-white">
+          <img src="/images/luminosity-home-logo.ico" alt="Home-logo"></img>
+        </a>
+        <span className="text-lg font-semibold font-sans hover:text-white cursor-pointer">
```

Comment:
I noticed that you've got `font-semibold` for the "Luminosity" text, but it looks like the Figma model has `regular`. If you want to change it up, just bring it up with the design team and see what they think. Also, I believe we want to use `font-inter` across the site, instead of sans-serif. Now that I think of it, we could probably just set it on some top-level component. Wanna see if you can make that work?

Reply:
Also, I you can probably increase the font size to match the proportions of the other text in the body. Check to see how this affects the padding, and try to dial that in to match the figma model.